docs/guide/contributing.rst

```diff
@@ -71,11 +69,28 @@ Prerequisites
    1. In macOS: Open the **Terminal** app.
    2. In Windows: From the Start Menu, open **Git Bash**.
 
-4. In the repo UI, find the file name corresponding to an `existing issue`_ or page you want to improve. 
+4. In the `Write the Docs www project`_, find the file name corresponding to an `existing issue`_ or page you want to improve. 
 
 For example, ``/docs/documentarians.rst`` creates https://www.writethedocs.org/documentarians/. 
 
-5. Review formatting guidelines for the Markup style the file uses. The example file above uses `reStructuredText (.rst) Markup`_. Other Markup styles for our guides include Markdown (.md).
+5. Review formatting guidelines for the Markup style the file uses. The example file above uses `reStructuredText (.rst)`_.
+
+.. _update-guide-via-github:
+
+Editing a single guide directly in the GitHub UI
+--------------------------------------------------
+
+1. :ref:`Complete the prerequisites <prerequisites>`.
+
+2. Click the pencil icon to edit this file in your fork of the www repo.
```

Comment:
```suggestion
2. Click the pencil icon to create a fork of the `www` repo and start editing the file in it.
```
Assuming it's the first edit (since otherwise, they probably don't need the guide).